Source/DevHelper/Template/Filter/Filter.cpp.in

```diff
@@ -15,7 +15,8 @@
 // -----------------------------------------------------------------------------
 //
 // -----------------------------------------------------------------------------
-@ClassName@::@ClassName@()@InitList@
+@ClassName@::@ClassName@() : 
+@InitList@
 {
```

Comment:
What if the InitList is empty? I think that the code itself should figure out if the InitList is empty and if it isn't empty then add the ':' character.

Reply:
InitList is never empty because by default it at least puts in AbstractFilter()

Reply:
Wait, never mind it can be empty.